code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -95,11 +96,6 @@
 	spawn(5)
 		eyeobj.loc = src.loc
 
-/mob/living/silicon/ai/Destroy()
-	eyeobj.ai = null
-	qdel(eyeobj) // No AI, no Eye

```

Comment:
qdel'ing the eyeobj has been moved into the single instance of ai/Destroy() here:https://github.com/ParadiseSS13/Paradise/pull/2310/files#diff-fdbfc27b5e83d4c741e4a11a99b94d44R232

where as nulling out the AI reference on the eyeobj is done here now: https://github.com/ParadiseSS13/Paradise/pull/2310/files#diff-c568c533b21b08e7112f217c88195b5dR32

It is very important, but it's still there =p